lib/iris/tests/unit/fileformats/nc_load_rules/actions/test__grid_mappings.py

```diff
@@ -755,6 +756,45 @@ def test_mapping__mismatch__nonll_coords_missing_system(self):
         )
         self.check_result(result, cube_no_cs=True, xco_stdname=False, yco_stdname=False)
 
+    def test_extended_mapping_basic_latlon(self):
+        # A basic reference example with a lat-long grid, but using extended
+        # grid mapping syntax on phenomenon.
+        #
+        # Rules Triggered:
+        #     001 : fc_default
+        #     002 : fc_provides_grid_mapping_(latitude_longitude)
+        #     003 : fc_provides_coordinate_(latitude)
+        #     004 : fc_provides_coordinate_(longitude)
+        #     005 : fc_build_coordinate_(latitude)
+        #     006 : fc_build_coordinate_(longitude)
+        # Notes:
+        #     * grid-mapping identified : regular latlon
+        #     * dim-coords identified : lat+lon
+        #     * coords built : standard latlon (with latlon coord-system)
+        result = self.run_testcase(phenom_grid_mapping="grid: yco xco")
+        self.check_result(result)
```

Comment:
Correct me if I'm wrong, but it does seem like `check_result()` does any checks for an expected coordinate system? More to do here?

Reply:
Small addition to assert a coord system actually exists: ff9041d2a